docs/modules/clients/pages/java-thin-client.adoc

```diff
@@ -0,0 +1,1118 @@
+= {java-standard-client-name}
+:url-cloud-signup: https://cloud.hazelcast.com/sign-up
+:page-toclevels: 3
+:description: The {java-standard-client-name} is a lightweight client that connects to the cluster and performs all operations through the connected node. It does not become a part of the cluster topology, never holds any data, and is not used as a destination for compute calculations. The {java-standard-client-name} supports simplified maintenance, a clear separation of client and cluster codebases, optimization of deployment, and opportunities for further optimization specific to the {java-standard-client-name}.
+[[java-client]]
+
+{description}
+
+== Get Started
+
+To get started, include the `hazelcast-java-client-{full-version}.jar` dependency in your classpath. Once included, you can start using this client as if
+you are using the Hazelcast API. The differences are discussed below.
+
+NOTE: If you have a Hazelcast {enterprise-product-name} license, you do not need to set the license key in your clients to use the xref:getting-started:editions.adoc#features-in-hazelcast-enterprise[{enterprise-product-name} features]. It is sufficient to set the license on the member side, and include the `hazelcast-enterprise-java-client-{full-version}.jar` dependency in your classpath.
+
+If using Maven, add the `hazelcast` dependency
+to your `pom.xml`, or to the `hazelcast-enterprise` dependency if you want the client to use {enterprise-product-name} features, and you have the Hazelcast {enterprise-product-name} license,
+which might have been done when starting using Hazelcast.
+
+The `pom.xml` file is updated as follows:
+
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>com.hazelcast</groupId>
+    <artifactId>hazelcast-java-client</artifactId>
+    <version>{full-version}</version>
+</dependency>
+----
+== Migrating from Embed client
+To migrate from the Embed client to {java-standard-client-name}, the only change required is in the `pom.xml` file:
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>com.hazelcast</groupId>
+    <artifactId>hazelcast-java-client</artifactId>
+    <version>{full-version}</version>
+</dependency>
+----
+The {java-standard-client-name} is mostly compatible with the Embed client. However, one notable difference is that methods which threw `UnsupportedOperationException` in the Embed client are not present at all in the {java-standard-client-name}. For example, the following methods from `ClientMapProxy`:
```

Comment:
Why is this different? Is there a decision log for this? We want 1:1 same with platform client including the exceptions we throw.

Reply:
As currently implemented: it is because the embed client shares some common code with the members and it stubs some methods via throwing exceptions in cases when the user decides to call impropriate method on the client side. 
In the Standard/Thin client the "member" code was eliminated.

Let's discuss this item with the developers outside the PR's conversation.